### ISSUE TITLE
NameError: undefined local variable or method `db'

### ISSUE THREAD

**Issue by beno** [educabilia/ohm-sorted#4]
When I run the tests I run into 20 errors like in the title. I get the feeling this was perhaps written for another Redis client library from what Ohm uses (redic)?

**Comment by federicobond** [comment]
Indeed. `ohm-sorted` supports only the 0.x and 1.x versions of ohm at the moment.
